spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/AbstractTraceChannelInterceptor.java

```diff
@@ -20,6 +20,9 @@
  */
 abstract class AbstractTraceChannelInterceptor extends ChannelInterceptorAdapter implements ExecutorChannelInterceptor {
 
+	/**
+	 * A default prefix for span name for discerning messaging origin of the span
```

Comment:
I have no idea what this means..  "discerning messaging origin of the span" 
do you mean that spans with this naming prefix came from messaging components? If so, maybe reinforce with an example? Because in this change it looks like this is a tag prefix, not a span name prefix, or maybe it is both?